src/main/java/com/saucedemo/pages/Login.java

```diff
@@ -48,13 +49,11 @@ public String getLoginLockOutError() {
         return lockOutError.getText();
     }
 
-    public void login(String user, String password) {
-
-        driver.get("https://www.saucedemo.com/");
-        setUsername(user);
+    public void login(String username, String password) {
+        driver.get(loginUrl);
```

Comment:
so this is where LOGIN_URL is better than loginUrl. Conventions for constants to be named.